src/patterns/data/default/entity_attribute.tsv

```diff
@@ -4100,3 +4100,291 @@ OBA:2050052	occipital lobe volume 	UBERON:0002021	occipital lobe	PATO:0000918	vo
+OBA:2060017	decreased cornea thickness	UBERON:0000964	cornea	PATO:0000915	thickness
+OBA:2060018	decreased trabecular bone thickness	UBERON:0002483	trabecular bone tissue	PATO:0000915	thickness
+OBA:2060019	thin cerebellar granule layer	UBERON:0002956	granular layer of cerebellar cortex	PATO:0000915	thickness
+OBA:2060020	thin hippocampus molecular cell layer	UBERON:0005368	hippocampus molecular layer	PATO:0000915	thickness
+OBA:2060021	thin hippocampus stratum oriens	UBERON:0005371	hippocampus stratum oriens	PATO:0000915	thickness
+OBA:2060022	thin hippocampus stratum radiatum	UBERON:0005372	hippocampus stratum radiatum	PATO:0000915	thickness
+OBA:2060023	thin tail	UBERON:0002415	tail	PATO:0000915	thickness
+OBA:2060024	thin ventricular wall	UBERON:0004784	heart ventricle wall	PATO:0000915	thickness
+OBA:2060025	dilated aorta	UBERON:0000947	aorta	PATO:0000117	size
+OBA:2060026	dilated heart left ventricle	UBERON:0002084	heart left ventricle	PATO:0000117	size
+OBA:2060027	mydriasis	UBERON:0001771	pupil	PATO:0000117	size
+OBA:2060032	decreased locomotor activity	NBO:0000013	locomotory behavior	PATO:0000161	rate
+OBA:2060037	hyporesponsive to tactile stimuli	NBO:0000332	touch related behavior	PATO:0000161	rate
+OBA:2060040	impaired pupillary reflex	NBO:0000412	pupillary light reflex	PATO:0000161	rate
+OBA:2060041	impaired righting response	NBO:0000578	righting reflex	PATO:0000161	rate
+OBA:2060042	abnormal digestive system physiology	UBERON:0001007	digestive system	PATO:0001509	functionality
+OBA:2060043	abnormal eye physiology	UBERON:0000970	eye	PATO:0001509	functionality
+OBA:2060044	abnormal lacrimal gland physiology	UBERON:0001817	lacrimal gland	PATO:0001509	functionality
+OBA:2060045	abnormal adipose tissue amount	UBERON:0001013	adipose tissue	PATO:0000070	amount
+OBA:2060046	abnormal abdominal wall morphology	UBERON:0003697	abdominal wall	PATO:0000051	morphology
+OBA:2060047	abnormal cerebellar granule layer morphology	UBERON:0002956	granular layer of cerebellar cortex	PATO:0000051	morphology
+OBA:2060048	abnormal coat/ hair morphology	UBERON:0001037	strand of hair	PATO:0000051	morphology
+OBA:2060049	abnormal hippocampal fimbria morphology	UBERON:0002310	hippocampus fimbria	PATO:0000051	morphology
+OBA:2060050	abnormal optic disk morphology	UBERON:0001783	optic disc	PATO:0000051	morphology
+OBA:2060051	abnormal vitreous body morphology	UBERON:0001798	vitreous body	PATO:0000051	morphology
+OBA:2060052	hydrops fetalis	UBERON:0007221	neonatal stage	PATO:0000025	composition
+OBA:2060053	increased body weight	UBERON:0000468	multicellular organism	PATO:0000125	mass
+OBA:2060054	increased brain weight	UBERON:0000955	brain	PATO:0000125	mass
+OBA:2060055	increased embryo weight	UBERON:0000922	embryo	PATO:0000125	mass
+OBA:2060056	increased epididymis weight	UBERON:0001301	epididymis	PATO:0000125	mass
+OBA:2060057	increased heart weight	UBERON:0000948	heart	PATO:0000125	mass
```

Comment:
The deduplication did not work:

```
OBA:VT0007028	heart mass	UBERON:0000948	heart	PATO:0000125	mass
```

Only deduplify on the `entity` and `attribute` columns!